doc/tech/roadmap.md

```diff
@@ -46,7 +48,17 @@ A Release is:
 Please refer to individual [release plan](./release_notes/) and features for detailed roadmap.
 
 <a name="2.3"></a>
-### Milestones
+### Detailed Milestones
+| Review | Milestone | Description | Activities | Comments |
+|--------|-----------|--------------|------------|---------|
+| Kick-Off | M0 | The goal of the release Kick-Off is to open the innovation platform for the intent to participate in the CNTT release. Release Kick-Off review takes place for each releases. | Identity a list of features and functionality will be documented and developed. | |
```

Comment:
what is the "innovation platform"?

The sentence "The goal of the release Kick-Off is to open the innovation platform for the intent to participate in the CNTT release" -- goal and intent -- can be simplified and the goal also has to be more than just indicating an intent to participate.

Typo: 
"Release Kick-Off review takes place for each releases" -- singular "release"
"Identity a list of features.." --- "Identify..."

Reply:
@pgoyal01 - Thanks for correcting the typo. This is indeed an innovation platform but there is no need to explain or define what is innovation platform here. Let me see if I can better reword it.